css/src/components/breadcrumbs.scss

```diff
@@ -12,7 +12,7 @@ $breadcrumb-separator: '\00200B/' !default;
 
 		&:not(.breadcrumbs-item-slashless)::after {
 			display: inline-block;
-			margin-inline: $breadcrumb-spacing-horizontal;
+			margin-inline-end: $breadcrumb-spacing-horizontal;
```

Comment:
Revert?

Reply:
Intention was right here, but looks like the incorrect spacing in Docs was not a symptom of any issues here in Atlas, just a hallmark of the default `white-space: normal`, which multiple white space chars into a single character. This PR doesn't need to merge after all. 😔 Apologies folks